### PR TITLE
change grade background color programmatically

### PR DESCRIPTION
**Pull request by jacqueline8711**
before: drawable files for every color

### CODE REVIEW

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/grades/ExamListAdapter.java [hunk 1]
```diff
@@ -21,24 +22,24 @@
...
-            .put("3,0", R.drawable.grade_3_0)
-            .put("3,3", R.drawable.grade_3_3)
-            .put("3,4", R.drawable.grade_3_3)
-            .put("3,7", R.drawable.grade_3_7)
-            .put("4,0", R.drawable.grade_4_0)
-            .put("4,3", R.drawable.grade_4_3)
-            .put("4,4", R.drawable.grade_4_3)
-            .put("4,7", R.drawable.grade_4_7)
-            .put("5,0", R.drawable.grade_5_0)
+    private static final Map<String, Integer> GRADE_COLOR = ImmutableMap.<String, Integer>builder()
```
Comment (thellmund) [inline app/src/main/java/de/tum/in/tumcampusapp/component/tumui/grades/ExamListAdapter.java]: Can we move this out of the ExamListAdapter and into Grade itself, e.g. exposing it via grade.getGradeColor(context)? Would help simplify the getView() method.

**Comment by thellmund** [comment]
LGTM! The comment above is just a suggestion.

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/grades/ExamListAdapter.java (hunk 1 above)
Comment (kordianbruck) [reply app/src/main/java/de/tum/in/tumcampusapp/component/tumui/grades/ExamListAdapter.java]: I like that suggestion - @jacqueline8711 could you quickly switch this up please?

**Comment by kordianbruck** [comment]
@jacqueline8711 ping. Please also resolve conflicts.

**Comment by pfent** [comment]
@kordianbruck please merge this one

**Pull request merged**